### DISCUSSION TITLE
from .color import rgb_to_yrgb?

### DISCUSSION THREAD

**Discussion by diverswan9** [nagadomi/nunif#234] [General]
`from .color import rgb_to_yrgb`  What is this for? I know the 'Y' relates to luminance.   

And thanks for the 'Drag-n-Drop' feature or has that always been there?

**Comment by nagadomi** [comment]
Have you encountered any errors?
I work on multiple tasks that are unrelated to iw3, so this repo have a lot of commits that are unrelated to iw3.

Drag and Drop feature is available from the initial version.
However, I did not describe about GUI in the English README, so most people might not have noticed it.
Japanese README describes it.

**Reply by diverswan9** [reply]
No errors... just curious and nosy. :)
